From f81a37f37f37e82ada1eb66735a978af761ae764 Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@oracle.com>
Date: Wed, 20 Nov 2024 17:14:49 -0800
Subject: [PATCH libXrender] Ensure all BufAlloc calls have trailing semicolon

Most already did, this fixes 3 outliers, so they build with
libx11@563ddd4ff5575e3f8bbf635cbdc339b09a7e285e

Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
Part-of: <https://gitlab.freedesktop.org/xorg/lib/libxrender/-/merge_requests/14>
(cherry picked from commit 6663858e918923d02c466298670c992a8437a17b)
---
 src/Glyph.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/Glyph.c b/src/Glyph.c
index 48e40c3..5148484 100644
--- a/src/Glyph.c
+++ b/src/Glyph.c
@@ -477,7 +477,7 @@ XRenderCompositeText8 (Display			    *dpy,
 	{
 	    int this_chars = nchars > MAX_8 ? MAX_8 : nchars;
 
-	    BufAlloc (xGlyphElt *, elt, SIZEOF(xGlyphElt))
+	    BufAlloc (xGlyphElt *, elt, SIZEOF(xGlyphElt));
 	    elt->len = this_chars;
 	    elt->deltax = xDst;
 	    elt->deltay = yDst;
@@ -588,7 +588,7 @@ XRenderCompositeText16 (Display			    *dpy,
 	    int this_chars = nchars > MAX_16 ? MAX_16 : nchars;
 	    int this_bytes = this_chars * 2;
 
-	    BufAlloc (xGlyphElt *, elt, SIZEOF(xGlyphElt))
+	    BufAlloc (xGlyphElt *, elt, SIZEOF(xGlyphElt));
 	    elt->len = this_chars;
 	    elt->deltax = xDst;
 	    elt->deltay = yDst;
@@ -694,7 +694,7 @@ XRenderCompositeText32 (Display			    *dpy,
 	{
 	    int this_chars = nchars > MAX_32 ? MAX_32 : nchars;
 	    int this_bytes = this_chars * 4;
-	    BufAlloc (xGlyphElt *, elt, SIZEOF(xGlyphElt))
+	    BufAlloc (xGlyphElt *, elt, SIZEOF(xGlyphElt));
 	    elt->len = this_chars;
 	    elt->deltax = xDst;
 	    elt->deltay = yDst;
-- 
2.54.0

